src/wrapper_generators.jl

```diff
@@ -34,13 +34,16 @@ macro generate_init_header(dependencies...)
                 isdefined($(dep), :PATH_list) && append!(PATH_list, $(dep).PATH_list)
                 isdefined($(dep), :LIBPATH_list) && append!(LIBPATH_list, $(dep).LIBPATH_list)
             end)
-            push!(eager_mode, :(isdefined($(dep), :eager_mode) && $(dep).eager_mode()))
+            depmod = getglobal(__module__, dep)
+            if isdefined(depmod, :eager_mode)
+                push!(eager_mode, :($(dep).eager_mode()))
```

Comment:
```suggestion
                push!(eager_mode, :($(dep).eager_mode()::Nothing))
```

Not really necessary since it's unused, but arguably good for hygiene